CMakeLists.txt

```diff
@@ -276,8 +276,11 @@ option(BUILD_MAIN_DUCKDB_LIBRARY
         TRUE)
 option(EXTENSION_STATIC_BUILD
         "Extension build linking statically with DuckDB. Required for building linux loadable extensions."
-        TRUE)
+        FALSE)
 
+if(WIN32)
+  set(EXTENSION_STATIC_BUILD TRUE)
```

Comment:
Why is this turned on by windows by default?

Reply:
We would have to revert the entire PR (including adding back support for delayload) to support disabling static builds on Windows again